### ISSUE TITLE
grouping of trait terms

### ISSUE THREAD

**Issue by megbalk** [obophenotype/bio-attribute-ontology#139]
general grouping class for 'bone' traits
general grouping class for 'morphology' traits

both will have 'characteristic of part of' rather than 'characteristic of' so that terms can be nested

As an example, currently 'neurocranium morphology trait' and 'facial skeleton morphology trait' are not nested (after inference) under 'skull morphology trait' even though both neurocranium and 'facial skeleton' are children of skull in UBERON

@rays22 can you help create a template and I can put in terms?

**Comment by cmungall** [comment]
Hmm the way I originally set up OBA these would be inferred to be part of.

It may seem odd to have part ofs but this is consistent with pato and the plant TO has always done it this way

The proposal loses precision

**Comment by megbalk** [comment]
When I run the reasoner for 'skull size trait' does not show up as an inferred SubClassOf for 'neurocranium size trait'

**Comment by matentzn** [comment]
@cmungall we have not made any changes to this side of OBA at all. It seems to have always use characteristic of! I am taking your comment as consent then :)

**Comment by matentzn** [comment]
@megbalk proposal is exactly to add the part ofs in you say you had already added, so I think your last comment is based on misunderstanding the proposal)

**Comment by megbalk** [comment]
ok, so I can go ahead and change some to "characteristic of part of"?

**Comment by matentzn** [comment]
yes go ahead @megbalk. Sparingly at first until we get a picture of what will change as a consequence of it.

**Comment by megbalk** [comment]
ok, I created a separate pattern called entity_attribute_partOf.yaml with data entity_attribute_partOf.tsv for 'characteristic of part of' using the neurocranium morphology trait as an example. It seems the entity_attribute.yaml does have a relation 'characteristic of part of', but I don't know how to tell the pattern how to decide which to use.

**Comment by matentzn** [comment]
> Don't know how to tell the pattern how to decide which to use.

Only one of the two relationships should be mentioned in the pattern. Whatever you reference in the `equivalentClass` section in the end of the pattern decides which relation is used. Yours should only use the `characteristic of part of` one.

Make a Draft PR so we can look at your branch, and make comments directly on it!

**Comment by rays22** [comment]
When the `BFO:0000050 part_of` relationships are materialised using ELK, then [neurocranium morphology trait](http://purl.obolibrary.org/obo/OBA_VT0000074) does show up as `'part of' some` [skull morphology trait](http://purl.obolibrary.org/obo/OBA_VT0000438)
  thanks to the GCI in the `entity_attribute` pattern:
```yaml
# (Q and characteristic_of some part-of some W) SubClassOf part-of some (Q and characteristic_of-in some W)
  69   │ # https://github.com/obophenotype/bio-attribute-ontology/issues/8
  70   │ logical_axioms:
  71   │   - axiom_type: GCI
  72   │     text: "%s and ('characteristic_of' some ('part_of' some %s)) SubClassOf 'part_of' some (%s and 'characteristic_of' some %s)"
```
![skull_morphology_trait2](https://user-images.githubusercontent.com/25171986/204643518-19ec17b3-4eba-4c4f-b7b2-2490b3eb6683.jpg)

Materialising the GCI seems to provide the desired groupings, does it not?

**Comment by matentzn** [comment]
Yes, but they will not be able to group the uPheno phenotypes correctly. We need to use the exact same pattern in uPheno and OBA if we want these two to interoperate..
